securepreferences/src/main/java/com/bitcoin/securepreferences/SecurePreferences.kt

```diff
@@ -72,12 +74,15 @@ class SecurePreferences(context: Context, private val namespace: String) {
     }
 
     fun getString(key: String): String? {
-        val pref: String? = mSharedPreferences.getString(key, null)
-        if (pref != null) {
-            val plaintext: String = mStringEncrypter.decryptString(pref)
-            return plaintext
+        //to achieve this for all decrypt method
+        //https://github.com/FlowCrypt/flowcrypt-android/commit/6b553f9f3de5e77802e016096cfc63605e58c7fe
+        synchronized(this){
```

Comment:
Kotlin has @Synchronized annotation that you can use.

Reply:
I think no problem as long as we use kotlin 1.3.x on this repo cause it's deprecated on 1.6, any way I updated.